docs/pages/develop/tools.mdx

```diff
@@ -49,6 +49,20 @@ To use EAS CLI, you need to install it globally on your local machine by running
 
 You can use `eas --help` in your terminal window to learn more about the available commands. For a complete reference, see [`eas-cli` npm page](https://www.npmjs.com/package/eas-cli).
 
+## Expo Doctor CLI
+
+Expo Doctor is a command line tool used to diagnose issues in your Expo project.
+
+To use Expo Doctor, run the following command in your project's root folder:
+
+<Terminal cmd={['$ npx expo-doctor']} />
+
+This command will perform checks and analyses on your codebase, looking for common issues, dependency compatibility, configuration files, and the overall health of your project. Once the check is complete, Expo Doctor will output the results.
+
+If Expo Doctor finds an issue, it will provide a description of the problem along with advice on how to fix it or where to find help.
```

Comment:
```suggestion
If Expo Doctor finds an issue, it provides a description of the problem along with advice on how to fix it or where to find help.
```

As per our [writing style guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#use-present-tense) let's prefer to use present tense here.